app/components/NoAppletsMessage.js

```diff
@@ -1,43 +1,56 @@
 import React from 'react';
 import { StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { colors } from '../theme';
-import { BodyText } from './core';
+import { BodyText, Hyperlink } from './core';
+import { joinExampleApplet } from '../state/applets/applets.thunks';
 
 const styles = StyleSheet.create({
   noAppletsContainer: {
-    position: 'absolute',
-    width: '100%',
-    height: '100%',
-    justifyContent: 'center',
-    alignItems: 'center',
+    padding: 50,
   },
   noAppletsMessage: {
     textAlign: 'center',
     color: colors.grey,
-    margin: 50,
+    fontSize: 18,
+  },
+  link: {
+    textAlign: 'center',
+    marginTop: 12,
+    fontSize: 18,
   },
 });
 
-const NoAppletsMessage = ({ isDownloadingApplets }) => (
+const NoAppletsMessage = ({ isDownloadingApplets, joinExampleApplet }) => (
   <View style={styles.noAppletsContainer}>
     {isDownloadingApplets
       ? (
         <BodyText style={styles.noAppletsMessage}>
-          Checking for studies...
+          Sychronizing...
```

Comment:
https://github.com/ChildMindInstitute/mindlogger-app/pull/159#discussion_r280900497 still relevant ― 'synchronizing' is missing its initial 'n'.